client/my-sites/upgrades/domain-management/style.scss

```diff
@@ -848,20 +848,18 @@ ul.email-forwarding__list {
 	}
 }
 
-.name-servers {
-	.icann-verification {
-		.button {
-			margin-bottom: 15px;
+.icann-verification {
```

Comment:
I move IcannVerification as a separate component in this PR, it is of equal importance with .name-servers. I can't imagine why would anybody implement another similar component called IcannVerification.